Parser/Data/graph.html

```diff
@@ -22,9 +24,9 @@
     id16([+]); id16 --> id14; id16 --> id15
     id17([*]); id17 --> id9; id17 --> id16
     id18([-]); id18 --> id5; id18 --> id17
-  </pre> 
-  <script type="module">
-    import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@10/dist/mermaid.esm.min.mjs';
-    mermaid.initialize({ startOnLoad: true });
-  </script>  
+  </pre>
+    <script type="module">
+        import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@10/dist/mermaid.esm.min.mjs';
+        mermaid.initialize({ startOnLoad: true });
+    </script>

```

Comment:
@gdineshkumar219 These spaces are not a part of this fix. Always check for unwanted lines and spaces before you commit your code.